boa_engine/src/context/mod.rs

```diff
@@ -683,22 +690,20 @@ impl Context {
     #[inline]
     pub fn execute(&mut self, code_block: Gc<CodeBlock>) -> JsResult<JsValue> {
         let _timer = Profiler::global().start_event("Execution", "Main");
-        let global_object = self.global_object().clone().into();
 
         self.vm.push_frame(CallFrame {
             prev: None,
             code: code_block,
-            this: global_object,
             pc: 0,
             catch: Vec::new(),
             finally_return: FinallyReturn::None,
             finally_jump: Vec::new(),
             pop_on_return: 0,
-            loop_env_stack: vec![0],
-            try_env_stack: vec![crate::vm::TryStackEntry {
+            loop_env_stack: Vec::from([0]),
+            try_env_stack: Vec::from([crate::vm::TryStackEntry {
                 num_env: 0,
                 num_loop_stack_entries: 0,
-            }],
+            }]),
```

Comment:
What is the benefit of using `Vec::from()`?

Reply:
I though it may generate more efficient code, but looking at the compiler output, it seems like it's exactly the same. Probably it's better for compile times, because rustc has to do less macro expansion.